legend-sdlc-model/src/main/java/org/finos/legend/sdlc/domain/model/project/configuration/ProjectDependency.java

```diff
@@ -98,7 +98,7 @@ public static ProjectDependency parseProjectDependency(String string, int start,
             throw new IllegalArgumentException("Invalid project dependency string: null");
         }
 
-        int delimiterIndex = string.indexOf(delimiter, start);
+        int delimiterIndex = string.lastIndexOf(delimiter);
         if ((delimiterIndex == -1) || (delimiterIndex >= end))
```

Comment:
Make sure that you're keeping within the start/end bounds.

```suggestion
        int delimiterIndex = string.lastIndexOf(delimiter, end - 1);
        if ((delimiterIndex == -1) || (delimiterIndex < start))
```